test/unit_tests/test_db.py

```diff
@@ -167,9 +168,10 @@ def test_autouse_fixture_rolls_back_bbb():
         ResourceTypeTable.query.filter(ResourceTypeTable.resource_type == "aaa").one()
 
 
+@pytest.mark.skip("TODO: Implement search method on SubscriptionSearchView")
 def test_full_text_search(generic_subscription_1):
     def get_subs_tsq(query: str) -> List[SubscriptionTable]:
-        subs = SubscriptionTable.query.search(query).all()
+        subs = SubscriptionSearchView.query.search(query).all()
         return subs
```

Comment:
I guess I would skip the assignment and just 'return  SubscriptionSearchView.query.search(query).all()'